src/main/kotlin/nl/knaw/huc/broccoli/resources/projects/ProjectsResource.kt

```diff
@@ -120,7 +120,13 @@ class ProjectsResource(
             context.read<Map<String, Any>>("$.hits.total")
                 ?.let { result["total"] = it }
 
-            extractAggregations(index, context)?.let { aggs.putAll(it) }
+            extractAggregations(index, context)?.let { aggsFromElastic ->
```

Comment:
Ik ken het data model van ES niet goed genoeg om meteen te doorgronden wat hier gebeurt... 
Aggs is een object met daarin per key een agg, en die agg is weer een lijst met keys en waardes, waarvan de keys mogelijk met de terms overeen komen?
Misschien zouden voor dit soort dingen een simpele unit test werken, met wat json erin en weer eruit?